app-modules/theme/src/Providers/ThemeServiceProvider.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Assist\Theme\Providers;
+
+use Filament\Panel;
+use Assist\Theme\ThemePlugin;
+use Illuminate\Support\ServiceProvider;
+
+class ThemeServiceProvider extends ServiceProvider
```

Comment:
If you could bring back the `boot()` with the empty morph map and the permission registry. Take a look at another Module for an example.

Those should be in all modules regardless and will be used a little bit later.